File: java/Car.java
class Car {
   private Integer id;
   private String license;
   private Account driver;
   private Integer passenger; 
   //METODO
   public Car(String license, Account driver) {
      //licence & driver son variables locales
      this.license = license;
      this.driver = driver;
   }
   void printDataCar(){
      if (passenger != null){
         System.out.println("License: " + license + " Driver: " + driver.name + " Passengers" +passenger);
      } else {
         System.out.println("Necesitas 4 pasajeros");
      }
   }
   // getters y setters
   public Integer getPassenger(){
      return passenger;
   }

   public void setPassenger(Integer passenger){
      if (passenger == 4){
         this.passenger = passenger;
      }else{
         System.out.println("Necesitas asignar 4 passajeros");
      }
   }
   public Integer getId() {
      return id;
   }
   public void setId(Integer id) {
      this.id = id;
   }
   public String getLicense() {
      return license;
   }
   public void setLicense(String license) {
      this.license = license;
   }
   public Account getDriver() {
      return driver;
   }
   public void setDriver(Account driver) {
      this.driver = driver;
   }

   

   

}

File: PHP/car.php
<?php
class Car{
    public $id;
    public $license;
    public $driver;
    public $passanger;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }
    
    public function printDataCar(){
        echo "
            license: $this->license
            conductor: $this->driver->name 
            document: $this->driver->document
            ";
    }
    /* For run this code execute:
    php -S localhost:8000 -t .
    and go to localhost in a web browser.
 */
    public function getPassenger(){
        return $this->passanger;
    }

    public function setPassenger($passenger){
        if ($passenger == 4){
            $this->passanger = $passenger;
        } else {
            echo "Necesitas asignar 4 pasajeros"; 
        }
    }
}